OloEngine/src/OloEngine/Scripting/Lua/LuaScriptEngine.cpp

```
@@ -83,34 +77,130 @@ namespace OloEngine
+        {
+            sol::protected_function_result createResult = onCreate.value()(entityID);
+            if (!createResult.valid())
+            {
+                sol::error error = createResult;
+                OLO_CORE_ERROR("[LuaScriptEngine] OnCreate error for entity {}: {}", entityID, error.what());
+            }
+        }
     }
 
-    void LuaScriptEngine::OnDestroyed(const Entity*)
+    void LuaScriptEngine::OnUpdateEntity(Entity entity, f32 ts)
```

Clang (14.0.6)

[warning] method 'OnUpdateEntity' can be made static

(readability-convert-member-functions-to-static)